jdaviz/models/physical_models.py

```diff
@@ -0,0 +1,332 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Models that have physical origins.
+"""
+# pylint: disable=invalid-name, no-member
+
+import warnings
+
+import numpy as np
+
+from astropy import constants as const
+from astropy import units as u
+from astropy.utils.exceptions import AstropyUserWarning
+from astropy.modeling.core import Fittable1DModel
+from astropy.modeling.parameters import Parameter
+
+__all__ = ["BlackBody"]
+
+
+class BlackBody(Fittable1DModel):
```

Comment:
I am guessing you copied this from astropy somewhere. Maybe add a dev comment about that?

Also, in the future when astropy/astropy#12304 and astropy/astropy#12318 are resolved, what is going to happen to this code?